canonical-web-and-design/juju.is
# Switch from node sass to sass

## Done
- Changed from node-sass to sass
- Changed from sass-lint to stylelint

## QA
- Lint the styles: `yarn lint-scss`
- Make sure the CSS builds: `yarn build-css`
- Run the site and check that it doesn't look broken

## Issue
Fixes #170
